### PR TITLE
FIX: Dont use hardcoded label

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/ac0xn6zfjcnv2jkz5fx5mh1ww8?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/7289)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/uugvudv0n578gz6m42dyg3utwr?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/7298)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/mudhz62nn2gwxjn6e5gf2xqc84?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/7299)

File: class/wallets/abstract-wallet.js [hunk 1]
```diff
@@ -213,7 +213,6 @@ export class AbstractWallet {
         this.secret = parsedSecret.ExtPubKey;
         const mfp = Buffer.from(parsedSecret.MasterFingerprint, 'hex').reverse().toString('hex');
         this.masterFingerprint = parseInt(mfp, 16);
-        this.setLabel('Cobo Vault ' + parsedSecret.MasterFingerprint);
```
Comment (Overtorment) [inline class/wallets/abstract-wallet.js:216]: do we have any other hints on what kind of HW wallet we are scanning from?

**Review by Overtorment — APPROVED** [review]

File: class/wallets/abstract-wallet.js (hunk 1 above)
Comment (marcosrdz) [reply class/wallets/abstract-wallet.js:216]: Not with URv2.

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/fp9bfkuwm2embe8g9dgh5defag?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/7313)

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**